tests/samples/songkick/elysianfields.html

```diff
@@ -28,6 +28,7 @@
     <meta property="og:type" content="songkick-concerts:artist">
     <meta property="og:title" content="Elysian Fields">
     <meta property="og:description" content="Buy tickets for an upcoming Elysian Fields concert near you. List of all Elysian Fields tickets and tour dates for 2017.">
+    <meta property="og:description" content="" />
```

Comment:
Maybe you could have the tests also check that whitespace characters, like spaces, are now ignored as well?

Reply:
Thank you! That's a good idea.

Reply:
@Gallaecio Do you want to take a look or I can just go ahead and merge this?
This is my first contribution to Extruct ;)